### DISCUSSION TITLE
Two pintaric users

### DISCUSSION THREAD

**Discussion by erlend-sh** [pintariching/rustle#45] [General]
![D499EE91-B7B3-4F35-8840-1C1697389C3F](https://user-images.githubusercontent.com/583842/189446260-7712f884-177b-42aa-93c4-57c34681085d.jpeg)

One of these accounts doesn’t appear to be properly picked up by GitHub, as it’s not reflected in the [Contributors](https://github.com/pintariching/rustle/graphs/contributors) graph.

**Comment by qwertydelle** [comment]
It happens when you dont have your email linked in git it just uses your whatever the current user is named

**Reply by pintariching** [reply]
Maybe because I merged some PRs in the mobile app? I'll check my settings thanks for the heads up!

**Comment by pintariching** [comment]
I figured it out. I had my name as user.name in my git config instead of my username. I think changing it fixed it, I did some commits from my pc and it shows my username now.
